app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java

```diff
@@ -132,6 +129,12 @@ public int snackBar(String message, String actionMessage, View.OnClickListener l
         return snackBar.message(message, actionMessage, listener, 999);
     }
 
+    public int snackBar(int resourceIdMessage, int resourceIdActionMessage, View.OnClickListener
```

Comment:
Use the `@StringRes` annotation for this and Android Studio will warn you if you accidentally pass the wrong type of value in here :)